.github/workflows/cve_scan.yaml

```diff
@@ -0,0 +1,35 @@
+name: CVE Scan
```

Comment:
I'm wondering if there is overlap with CodeQL scanning already setup for this repository.

Reply:
@avi-08 where in the security tab will there results show up?

Reply:
@vuil The reports can be accessed from Security > Code Scanning > PR# filter. For this PR: https://github.com/vmware-tanzu/tanzu-framework/security/code-scanning?query=pr%3A4530+is%3Aopen, it shows 0 open and 18 closed issues

Reply:
non-blocking: If we can add an automated comment on this PR with a link mentioned above so we can view the alerts easily, that would be great.

Reply:
Added the comment step in workflow